### PR TITLE
Updated with changes from bitbucket private repo and original repo, added github actions for npm publishing

### PR DESCRIPTION
**Pull request by rafael-scanreco**
The publishing was tested here: https://github.com/scanreco/node-lz4/actions/runs/11817184421

I bump up the minor version once more and changed the .yml to only run on creating a release for after this PR is merged.

https://www.npmjs.com/package/@scanreco/node-lz4

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,30 +2,30 @@
...
   "description": "LZ4 streaming compression and decompression",
   "keywords": [
     "lz4",
     "compression",
     "decompression",
     "stream"
   ],
-  "version": "0.6.3",
-  "homepage": "http://github.com/pierrec/node-lz4",
+  "version": "0.6.5",
```
Comment (jubesab) [inline package.json]: Since this is not backward compatible, shouldn't we bump to a major version?

**Review by jubesab — COMMENTED** [review]
Would it be possible to remove the mention of Electron in the commit message since it's not related to that particular change?

**Comment by rafael-scanreco** [comment]
> Would it be possible to remove the mention of Electron in the commit message since it's not related to that particular change?

I have reworded the commit

File: package.json (hunk 1 above)
Comment (rafael-scanreco) [reply package.json]: 0.7.0 or straight to 1.0.0?

**Review by jubesab — APPROVED** [review]

**Pull request merged**